test/suites-ostree/fdo.robot

```diff
@@ -28,6 +28,7 @@ FIDO Onboarding Device
     ...    MicroShift data, unhealthy stored in health file,
     ...    and a deployment gap (no-microshift rollback ).
 
+    Mask Grub Boot Success Timer
```

Comment:
Should we build this into the image blueprints or kickstart file?

Reply:
Not entirely sure, but I doubt blueprints have this capabilities. [They can enable/disable services](https://www.osbuild.org/guides/image-builder-on-premises/blueprint-reference.html#systemd-services), but it's not a service and it cannot be disabled, only masked. Furthermore, it's user-specific (--user) - notice no `sudo`. [0]

If we want to include it, my bet would on be kickstart but my internal paranoia would at least keep a check if this is not masked, so we don't run into this problem by accident wondering what's happening like I did.

----
[0]

```
$ systemctl --user disable grub-boot-success.timer
The unit files have no installation config (WantedBy=, RequiredBy=, Also=,
Alias= settings in the [Install] section, and DefaultInstance= for template
units). This means they are not meant to be enabled or disabled using systemctl.
 
Possible reasons for having this kind of units are:
• A unit may be statically enabled by being symlinked from another unit's
  .wants/ or .requires/ directory.
• A unit's purpose may be to act as a helper for some other unit which has
  a requirement dependency on it.
• A unit may be started when needed via activation (socket, path, timer,
  D-Bus, udev, scripted systemctl call, ...).
• In case of template units, the unit is meant to be enabled with some
  instance name specified.
```

on the other hand, masking seems to be just symlinking to /dev/null:
```
$ systemctl --user mask grub-boot-success.timer
Created symlink /home/redhat/.config/systemd/user/grub-boot-success.timer → /dev/null.
```

Reply:
OK, maybe leaving it in the test code is the best way to be explicit that we need to take the step.